library/spdm_crypt_lib/fips/libspdm_selftest_aes_gcm.c

```diff
@@ -0,0 +1,146 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "internal/libspdm_crypt_lib.h"
+
+#if LIBSPDM_FIPS_MODE
+
+/**
+ * AES_GCM self_test: AES_GCM KAT test and AES_GCM PCT test.
```

Comment:
This is confusing - please remove the comment for "PCT test".

Reply:
Thanks. I have deleted the "PCT test" comment.